speech/setup.py

```diff
@@ -52,7 +52,6 @@
 REQUIREMENTS = [
     'google-cloud-core >= 0.20.0',
     'gapic-google-cloud-speech-v1beta1 >= 0.11.1, < 0.12.0',
-    'grpc-google-cloud-speech-v1beta1 >= 0.11.1, < 0.12.0',
```

Comment:
@dhermes the gapic one brings in the grpc one, right? In that case, I'd avoid specifying the transitive dependency.